go.mod

```diff
@@ -27,6 +27,7 @@ require (
 	github.com/coreos/go-systemd/v22 v22.5.0
 	github.com/cpuguy83/go-md2man/v2 v2.0.2
 	github.com/cyphar/filepath-securejoin v0.2.4
+	github.com/danwinship/knftables v0.0.12
```

Comment:
Discussed [here](https://github.com/kubernetes/kubernetes/pull/121046#discussion_r1367748528). I do plan to move it to k-sigs. Antonio and I agreed that should happen before beta.

The reason for not putting it in k/k is that it will be used / is already being used by code outside of k/k. (Some links in the linked comment.)